clients/tabby-agent/tests/server.docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.5"
```

Comment:
1. do not checkin this docker compose.
2. test against StarCoder-1B instead of SantaCoder-1B